config/maps.txt

```diff
@@ -44,6 +44,6 @@ endmap
 
 map shipstation
 	minplayers 0
```

Comment:
Increase this and the maximum player count to 200 to ensure that no one has to suffer playing on this map until it's actually feature-complete and vaguely playable.

Reply:
The iron repath fucked the PR somehow, I suspect due to something that happened when gandalf merged master back into it. I wasn't aware of it because I was working on several other PRs. Kneejerk reactions aren't going to help. I've already got a fix ncoming.